spec/controllers/authentication_controller_spec.rb

```diff
@@ -13,11 +13,29 @@
     it 'creates a new person' do
       expect { subject }.to change { Person.count }.by(1)
     end
-    it 'redirects the person to the root url' do
-      expect(subject).to redirect_to(root_url)
-    end
+
```

Comment:
Notice you are putting whitespace above your it blocks. If you are going to adopt a different formatting style than what the file was previously following can you please update all the other occurrences throughout the file?

hint - *lines 12 - 13*